docs/tutorial_human_popgen.md

```diff
@@ -0,0 +1,515 @@
+# Tutorial on how to set up nf-core/eager for metagenomic screening
```

Comment:
@TCLamnidis check title. I know you are jelly that I work on the cool stuff, but I think it might confuse other poor poop-genners ;).